tests/integration/security/testdata/rbac/v1beta1-negative-match.yaml.tmpl

```diff
@@ -0,0 +1,132 @@
+# The following policy denies access to path with prefix "/prefix" except "/prefix/whitelist" to workload b
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-b-deny
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "b"
+  action: DENY
+  rules:
+  - to:
+    - operation:
+        paths: ["/prefix*"]
+        notPaths: ["/prefix/whitelist"]
+---
+
+# The following policy denies access from other namespaces
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-c-same-namespace
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "c"
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notNamespaces: ["{{ .Namespace }}"]
+---
+
+# The following policy denies access to workload d if it's not mTLS, in other words,
+# it allows only mTLS traffic to access workload d
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-d-mtls-traffic
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "d"
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notPrincipals: ["*"]
+---
+
+# The following policy enables mTLS with PERMISSIVE mode for workload b, c and d
+
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: "permissive"
+  namespace: "{{ .Namespace }}"
+spec:
+  targets:
+  - name: b
```

Comment:
do we have to use targets? This is about to be deprecated. @diemtvu

Reply:
can we construct tests to avoid referencing this field?

Reply:
done